gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/java/io/gravitee/am/gateway/handler/root/resources/handler/user/PasswordPolicyRequestParseHandler.java

```diff
@@ -54,7 +54,7 @@ public void handle(RoutingContext context) {
         Optional<PasswordSettings> passwordSettings = PasswordSettings.getInstance(client, this.domain);
 
```

Comment:
In bug, I can see
"Versions affected:
4.1.x
4.2.x"

why is this PR to 4.0.x?

Reply:
it is also present in 4.0

Reply:
ok, please update Jira and public bug then